https://sourceforge.net/p/zbar/patches/37/

fix from Debian for crashes when importing the python module.
http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=702499

this doesn't happen on some arches as the data naturally ends up with zero
data after the structure, but on some (like arm), it isn't so we crash when
python walks the list.

--- a/imagescanner.c
+++ b/imagescanner.c
@@ -68,6 +68,7 @@ imagescanner_get_results (zbarImageScanner *self,
 
 static PyGetSetDef imagescanner_getset[] = {
     { "results", (getter)imagescanner_get_results, },
+    { NULL },
 };
 
 static PyObject*
